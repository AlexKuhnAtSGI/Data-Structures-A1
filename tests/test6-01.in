one
two
three
four
five
six
seven
eight
nine
ten
ten
ten
ten
nine
four
five
six
seven