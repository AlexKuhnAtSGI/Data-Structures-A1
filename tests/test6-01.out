one
six
ten
two
five
four
nine
eight
seven
three
